pinot-common/src/main/java/org/apache/pinot/common/function/TransformFunctionType.java

```diff
@@ -87,7 +87,10 @@
 
   // Geo relationship
   ST_CONTAINS("ST_Contains"),
-  ST_EQUALS("ST_Equals");
+  ST_EQUALS("ST_Equals"),
+
+  // Geo indexing
+  geoToH3("geoToH3");
```

Comment:
We need all capital case without underscore to make the `getTransformFunctionType()` work
```suggestion
  GEOTOH3("geoToH3");
```